app/api/items/add_item.go

```diff
@@ -112,6 +156,50 @@ func (in *NewItemRequest) canCreateItemsRelationsWithoutCycles(store *database.D
 	return count == 0
 }
 
+// swagger:operation POST /items items itemCreate
+// ---
+// summary: Create an item
+// description: >
+//
+//   Creates an item with parameters from the input data with `items.idDefaultLanguage` = `language_id`.
+//   Also it
+//
+//     * inserts a row into `users_items` with given `language_id`, `title`, `image_url`, `subtitle`, `description`,
+//
+//     * creates a new `groups_items` row with: `idItem` = `items.ID`, `idGroup` = `idGroupSelf` of the current user,
```

Comment:
Let's give the meaning instead (keep the tech details in parenthesis if you think it is relevant): "give full access to the item to the current user".

Reply:
fixed